new_platforms/vsextension/ProjectTemplates/oeenclave/MyEnclave.vcxproj

```diff
@@ -17,14 +17,26 @@
       <Configuration>Debug</Configuration>
       <Platform>Win32</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|ARM">
+      <Configuration>Debug</Configuration>
+      <Platform>ARM</Platform>
+    </ProjectConfiguration>
     <ProjectConfiguration Include="Prerelease|Win32">
       <Configuration>Prerelease</Configuration>
       <Platform>Win32</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Prerelease|ARM">
+      <Configuration>Prerelease</Configuration>
+      <Platform>ARM</Platform>
+    </ProjectConfiguration>
     <ProjectConfiguration Include="Release|Win32">
       <Configuration>Release</Configuration>
       <Platform>Win32</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|ARM">
+      <Configuration>Release</Configuration>
```

Comment:
With ARM (OP-TEE) release there is the same issue as with SGX release: we don't have a mechanism for release signing. The matter is somewhat simpler for OP-TEE in the sense that you can just set the signing key to the one you want to use and there is no third-party involved (i.e., Intel). However, we still don't have the policy mechanism to tell OP-TEE what key to check against. As it stands, the TA Dev Kit uses the debug signing key.

Perhaps we should clarify the documentation regarding this. There is only one note about SGX release signing in [Windows Development](https://github.com/Microsoft/openenclave/blob/feature.new_platforms/new_platforms/docs/win_sgx_dev.md#build-configurations), but no corresponding note for OP-TEE. There is no mention of release signing limitations in the other documents as far as I've seen.

Reply:
Yes, just cover in docs for now, thanks